.github/CONTRIBUTING_GUIDELINES.md

```diff
@@ -31,6 +31,13 @@
 
 - Install `nodejs` and `npm` on your local machine. For [windows](https://www.geeksforgeeks.org/installation-of-node-js-on-windows/), [linux](https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-20-04) and [mac](https://nodesource.com/blog/installing-nodejs-tutorial-mac-os-x/)
 
+- Install `Visual Studio` on your local machine, only for [windows](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools). (Please note that this is different from Visual Studio Code)
+- Mac useer install `Xcode Command Line Tools` by running
+
+  ```git
```

Comment:
Why 'git'? It should be 'bash' or something similar (that markdown supports)

Reply:
@Sakshisaraswat23 All occurrences of \```git, \```pnpm, \```docker etc need to be replaced with \```sh.